frontend/package.json

```diff
@@ -2,10 +2,12 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:8080",
   "dependencies": {
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.7.2",
```

Comment:
looks like we don't need this right now since you're not using it yet. also I think the version of node you guys are on will have `fetch` already natively built in, so you don't really need axios at all

Reply:
Removed.